canvas_modules/common-canvas/src/common-properties/controls/controls.scss

```diff
@@ -42,6 +42,7 @@
 	padding-bottom: $spacing-05; // Adds padding to the control
 	&.action {
 		display: flex;
+		justify-content: space-around;
```

Comment:
Does this fix the issue? I don't see it fixed when I download the PR.

Reply:
It doesn't fix the issue
![Screenshot 2024-12-26 at 11 18 09 AM](https://github.com/user-attachments/assets/3dd560ff-678e-4e52-ae82-28a75169e003)